apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -11,6 +14,29 @@ class AuthService {
 		this.userService = userService;
 	}
 
+	public async signIn(
+		userRequestDto: UserSignInRequestDto,
+	): Promise<UserSignInResponseDto> {
+		const { email, password } = userRequestDto;
+		const user = await this.userService.find({ email });
+
+		if (!user) {
+			throw new HTTPError({
+				message: "User not found",
+				status: HTTPCode.NOT_FOUND,
+			});
+		}
+
+		if (user.toNewObject().passwordHash !== password) {
```

Comment:
You should hash a password and compare hashes. User will never send hashed value